content/events/2019-11-12-game-development-javascript.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Game Development with JavaScript"
+type: event
+date: 2019-11-12
+starttime: 18:00
+endtime: 21:00
+presenter: Katie Walker
+presenter_url: https://kat.codes/
+presenter_img: https://github.com/katiew812.png
+presenter_bio: |
+  Katie is a recent Computer Science and Mathematics graduate of the University of Sheffield and currently works at Capital One as a Software Development Engineer. She worked previously as Game Developer at Tombola and now enjoys creating games and attending hackathons and conferences in her spare time.
+github_url: https://github.com/katiew812
+twitter_url: https://twitter.com/KatCodes
+meetup_url: https://www.meetup.com/NottsJS/events/vjnvhryzpbqb/
+---
+
+The most common use of JavaScript for game development is to recreate a classic game of Pong or Breakout, often to practice the language. However, there are many more possibilities within game development than just plain HTML and JavaScript and many are fantastic for developing new skills or for a fun side project. In this talk, I'll be discussing various techniques of developing 2D and 3D games for web, mobile and even smartwatch. I'll be covering general concepts of development and touching on various frameworks to enhance these different areas, such as Pixi.js and Phaser.js, which can transform a simple 2D browser game to a full-fledged mobile or desktop game. There will also be additional mentions of integrating technologies already well known, such as React and Vue.js, into our games to make the jump into game development less of a learning curve than you may think.
```

Comment:
This is pretty noisy on the website. Could do with some paragraph breaks.
However, let's post it and clean up later